src/js/content-script.js

```diff
@@ -6,7 +6,17 @@ function makeSetAudioURL(videoElement, url) {
       vid = vid.substring(0, pos);
     }
 
-    var bgUrl = `https://img.youtube.com/vi/${vid}/0.jpg`;
+    var maxresthumb = new Image();
+    maxresthumb.url = `https://img.youtube.com/vi/${vid}/maxresdefault.jpg`;
+    
+    var bgUrl;
+    if (maxresthumb.width = 120) {
```

Comment:
There's issue in this line^

Also can you please check what would be width in case youtube return 404 ? 

Thanks for the fix!

I have cherry-picked your changes related to reloading page.